src/SFA.DAS.CourseDelivery.Domain.UnitTests/Models/WhenMappingFromProviderToProviderSummary.cs

```diff
@@ -25,7 +25,7 @@ public void Then_The_Fields_Are_Correctly_Mapped(Provider source)
         [Test]
         public void And_Null_Then_Returns_Null()
         {
-            var actual = (ProviderSummary)null;
```

Comment:
?

Reply:
Might be clearer splitting it slightly, but the original (ProviderSummary)null wasn't invoking the implicit operator, it was just setting (ProviderSummary) as null